Src/CSharpier/SyntaxPrinter/Token.cs

```diff
@@ -62,7 +62,7 @@ public static Doc PrintWithSuffix(SyntaxToken syntaxToken, Doc suffixDoc)
                 && syntaxToken.Text.StartsWith("@"))
                 || (syntaxToken.Kind() == SyntaxKind.InterpolatedStringTextToken
                 && syntaxToken.Parent!.Parent
-                    is InterpolatedStringExpressionSyntax { StringStartToken:  { RawKind: (int)SyntaxKind.InterpolatedVerbatimStringStartToken }  } )
+                    is InterpolatedStringExpressionSyntax { StringStartToken: { RawKind: (int)SyntaxKind.InterpolatedVerbatimStringStartToken } })
```

Comment:
Is there a bug tracking breaking these?

Reply:
I believe this is the same problem https://github.com/belav/csharpier/issues/214